lib/fluent/plugin/base.rb

```diff
@@ -70,6 +76,19 @@ def multi_workers_ready?
         true
       end
 
+      def acquire_worker_lock(name, &block)
+        if fluentd_lockdir.nil?
+          raise RuntimeError, "fail to create lockfile on '#{fluentd_lockdir}'"
```

Comment:
```suggestion
          raise RuntimeError, "failed to create lock file becuase FLUENTD_LOCKDIR isn't set'"
```

It may be better that we use an appropriate exception class instead of a general propose `RuntimeError`.

Reply:
If using `RuntimeError` after all, then should we write it just as follows?

```ruby
raise "fail to create lock file because FLUENTD_LOCKDIR isn't set'"
```

Reply:
Fixed in a538b9bcc6561aac500823f1b7332edae8c2d0cb.

>It may be better that we use an appropriate exception class instead of a general propose RuntimeError

I defined "InvalidLockDirectory" error for that purpose.